EZP-32199 - SF documentation change (#89)
ezsystems/user-documentation: docs/site_organization/site_factory.md

    
    If all required fields are filled out you can select **Create,** and the site will be added to the site list in the **Site** tab.
    
    !!! note

>     A SiteAccess that you create in SiteFactory is always treated with lower priority than a SiteAccess 

Reviewer: Indent missing in the whole note

>     A SiteAccess that you create in SiteFactory is always treated with lower priority than a SiteAccess 

Reviewer: ```suggestion
    A SiteAccess that you create in Site Factory is always treated with lower priority than a SiteAccess 
```

>     defined by the administrator as part of the [configuration procedure](https://doc.ibexa.co/en/latest/guide/siteaccess/#configuring-siteaccesses). 

Reviewer: ```suggestion
    defined by the administrator as part of [configuration](https://doc.ibexa.co/en/latest/guide/siteaccess/#configuring-siteaccesses). 
```

>     For example, if you create a site that uses the `fr` path in SiteFactory, and the administrator 

Reviewer: ```suggestion
    For example, if you create a site that uses the `fr` path in Site Factory, and the administrator 
```
